CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
 # Braintree Android SDK Release Notes
 
 ## unreleased
-
+* PayPal
+  * Send `is_vault` in `event_params` to PayPal's analytics service (FPTI)s
 * Venmo
-  * Send `link_type` in `event_params` to PayPal's analytics service (FPTI)
+  * Send `link_type` and `is_vault` in `event_params` to PayPal's analytics service (FPTI)
```

Comment:
Can we update these to just say send `link_type` and `is_vault` analytics? - we don't need to state internal service naming.

Reply:
Addressed: [023cad0](https://github.com/braintree/braintree_android/pull/1004/commits/023cad0c49be7e885155fc1cee7c4351f9b99694)